tests/runner.py

```diff
@@ -37,6 +37,8 @@ def path_from_root(*pathelems):
 from tools.shared import *
 from tools.line_endings import check_line_endings
 
+logger = logging.getLogger('tests/runner.py')
```

Comment:
Why?

Can multiple subprocess reuse the same logger? A comment would help I think

Reply:
I'm still not totally clear on why logger vs logging

Reply:
logging.xxxx is calling methods on the root logger, which is always called "root".. which means all log messages have the word root in them.   When you have larger programs or programs with which invoke each other (as in this case) is better to have distinct loggers.

The existing code which imported the logging module from the shared module is just plane weird.

Reply:
Ah, makes sense
Yeah that was weird